tests/test_reading.py

```diff
@@ -94,14 +95,14 @@ def test_check_heif_magic(magic):
 
 
 def test_check_heif_magic_wrong():
-    assert not check_heif_magic(b'    ftyphey!    ')
+    assert not check_heif_magic(b'    fytphey!    ')
```

Comment:
?

Reply:
It becomes maybe in the mew version. We can’t skip “maybe”, it’s valid images, so just test that some garbage doesn’t allowed